Autopilot/PathManager/Inc/pathManager.hpp

```diff
@@ -0,0 +1,24 @@
+// #pragma once is used to ensure that you don't include the same header file more than once?
+#pragma once
+// including the pure abstract class that we will define later on
+#include "pathManagerStateManager.hpp"
+
+//this is the pure abstract class... but why are we just writing the name of the class here?? !QUESTION FLAG!
```

Comment:
I answered this in our DM, are you still confused about it ? It's the whole circular dependency thing

Reply:
oops sorry these are comments I forgot to take out